booster/test/rpc-integration/test-vacuous/response-vacuous-but-rewritten.json

```diff
@@ -3,7 +3,7 @@
     "id": 1,
     "result": {
         "reason": "vacuous",
-        "depth": 1,
+        "depth": 0,
```

Comment:
The test name is not quite right any more  (the state is _not_ rewritten). 
The difference to the other test is that a rewrite `b -> d` would be possible, so maybe `vacuous-not-rewritten` ?